oluseyianani/React-App-Boilerplate
# #165485021 optimize build for production

### What does this PR do?
- optimize build for the production environment

### Description of Task to be completed?
- minifies the `.bundle.js` file on production
- add jest and enzymes configuration for testing

### How should this be manually tested?
 - run `npm test` to mount the initial test successfully
- run `npm run prod` to minify `.bundle.js` file for the production env.
- run `npm run dev` to build file for the development environment.
- You will see the changes on the `dist/` folder.

### Any background context you want to provide?
n/a

### What are the relevant pivotal tracker stories?
[#165485021](https://www.pivotaltracker.com/story/show/165485021)

### Screenshots (if appropriate)
n/a
